### ISSUE TITLE
Loading just the discriminator model

### ISSUE THREAD

**Issue by slala2121** [mit-han-lab/data-efficient-gans#51]
Hi,

I tried loading the discriminator model and evaluating on some images but was running into some issues (don't have the error message handy right now but will try to update when I rerun the code), but I'm wondering if there's anything amiss with how I'm loading it:

`with tf.device('/gpu:0'):
            print('Constructing networks...')
            
            D_neg = tflib.Network('D', num_channels=num_channels, resolution=resolution, label_size=label_size, **D_args)
            resume_networks = misc.load_pkl(resume_pkl_neg)
            rG, rD, rGs = resume_networks
            D_neg.copy_vars_from(rD)

            dl=torch.utils.data.DataLoader(neg_dataset, batch_size=batch_size)
            d_scores_neg=[]
            for batch_index, (img, label) in enumerate(dl):
                # d_out_neg = D_neg.run(img.numpy(), is_training=False, minibatch_size=batch_size)
                with sess.as_default():
                    d_out_neg = D_neg.get_output_for(tf.convert_to_tensor(img.numpy()), is_training=False).eval()
                d_scores_neg.extend(list(d_out_neg))
            d_scores_neg=np.array(d_scores_neg)`

Thanks.

**Comment by slala2121** [comment]
Resolved. I was getting an error with the input dimensions.
